src/paho/mqtt/client.py

```diff
@@ -1794,7 +1794,7 @@ def loop_start(self):
 
         self._sockpairR, self._sockpairW = _socketpair_compat()
         self._thread_terminate = False
-        self._thread = threading.Thread(target=self._thread_main)
+        self._thread = threading.Thread(target=self._thread_main, name=self._client_id.decode())
```

Comment:
```suggestion
        self._thread = threading.Thread(target=self._thread_main, name=f"paho-mqtt-client-{self._client_id.decode()}")
```

I think the thread name should include that it's coming from paho. The client_id may not include the fact that it's a MQTT client and the program may not only be a MQTT client.